application/forms/Command/Object/AcknowledgeProblemForm.php

```diff
@@ -21,6 +23,23 @@ class AcknowledgeProblemForm extends CommandForm
 {
     use Auth;
 
+    public function __construct()
+    {
+        $this->on(self::ON_SUCCESS, function () {
+            $countObjects = count($this->getObjects());
+            $objectsType = 'services';
+            if (current($this->getObjects()) instanceof Host) {
+                $objectsType = 'hosts';
```

Comment:
No variables for translatable strings. Use this control structure to define the entire message.

Reply:
Oh and please perform this differentiation everywhere (my preferred solution) or nowhere. Currently it's only done here.

Reply:
Not sure 🤔 whether you''ve seen it in hindsight but have used it pretty much everywhere I can.